app/src/main/java/com/lightricks/feedexercise/data/Repository.kt

```diff
@@ -0,0 +1,9 @@
+package com.lightricks.feedexercise.data
+
+import io.reactivex.rxjava3.core.Completable
+import io.reactivex.rxjava3.core.Observable
+
+interface Repository {
```

Comment:
Repository is too general. I would name it as before `FeedRepository` and the implementation since we only have one I would name it `FeedRepositoryImpl`.